client-participation/gulpfile.js

```diff
@@ -76,10 +77,10 @@ var host;
 var basepath_visbundle_dev = "/foo";
 
 var webpack = require("webpack");
-var config = require("./webpack.config.dev");
-config.output.publicPath = basepath_visbundle_dev + "/js/";
-// console.log(config.output.publicPath);
-var compiler = webpack(config);
+var webpack_config = require("./webpack.config.dev");
```

Comment:
@patcon  (cc: @tamas-soos-toptal), 

Should we merge webpack config into the convict schema?

Reply:
After thinking about this a bit, I think we should consider the webpack config to be "implementation details". Someone setting up an instance is unlikely to want to mess with _most_ of these settings. However, if we find that there are settings we _do_ want to expose from the main config, we can probably munge those in (as we do with `output.publicPath`). Does that make sense?